### DISCUSSION TITLE
脚本失效，已经更换过源，依然不行。

### DISCUSSION THREAD

**Discussion by Nothingness-Void** [the1812/Bilibili-Evolved#4184] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/55913486/236655383-1851defc-9f69-408b-9595-2adb33a102ba.png)
主页就这样，搜索栏也没了，脚本也无法正常生效

f12打开控制台后报错为
`The resource https://s1.hdslb.com/bfs/static/laputa-home/client/assets/login-guide.68b48c4c.png was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.`

**Comment by Geeyun-JY3** [comment, marked as answer]
每个版本都有人汇报这问题：https://github.com/the1812/Bilibili-Evolved/discussions/4095#discussioncomment-5505418

还是建议去和相关 issue 比对配置，如有必要跟帖回复补充你的配置。

**Reply by Nothingness-Void** [reply]
感谢大佬回复，我换了台电脑（相同网络环境，相同插件版本和源，相同浏览器及版本）就可以了，我再研究研究
